prow/config/jobs.go

```diff
@@ -169,6 +169,8 @@ type Periodic struct {
 	Tags []string `json:"tags,omitempty"`
 	// Run these jobs after successfully running this one.
 	RunAfterSuccess []Periodic `json:"run_after_success,omitempty"`
+	// Maximum number concurrent executions of this jobs, unset is 1, 0 is infinite.
+	MaxConcurrency *int `json:"max_concurrency,omitempty"`
```

Comment:
I'll make the change.  Had a couple of questions first though.

1) Should I add an accessor that returns a default (0) to `JobBase` so we don't need to `nil` check everywhere?
2) Should I set the defaults in the config parse and then assume non-nil on usage since there should be no nil values: https://github.com/kubernetes/test-infra/blob/master/prow/config/config.go#L1124-L1170